src/components/ledger/LedgerConnector.tsx

```diff
@@ -110,12 +116,42 @@ export function LedgerConnector({
     [capture, getAvaxBalance]
   );
 
+  const isLedgerWalletExist = useCallback(
+    async ({ xpub, xpubXP, publicKeys, derivationPath }) => {
+      setIsLedgerExistsError(false);
+      try {
+        return await importLedger({
+          xpub,
+          xpubXP,
+          pubKeys: publicKeys,
+          secretType:
+            derivationPath === DerivationPath.BIP44
+              ? SecretType.Ledger
+              : SecretType.LedgerLive,
+          dryRun: true,
+        });
+      } catch (e) {
+        setIsLedgerExistsError(true);
+        throw new Error('wallet already exists');
```

Comment:
We should check the error message - there are other reasons `importLedger(...)` could fail.